pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -52,10 +52,21 @@ type k8sPluginConfig struct {
 type podInfo struct {
 	// We only care about namespace, serviceAccountName and containerID
 	Metadata struct {
-		Namespace string `json:"namespace"`
+		UID             string            `json:"uid"`
+		Namespace       string            `json:"namespace"`
+		Labels          map[string]string `json:"labels"`
+		OwnerReferences []struct {
+			Kind string `json:"kind"`
+			Name string `json:"name"`
```

Comment:
Might be nice to have the uid here too, as an alternative to the name

Reply:
We'd need distinct selectors for name v.s. uid. Curious on what formats people think are resonable. Maybe something like:

```
pod-owner-name:DaemonSet:blog
pod-owner-uid:DaemonSet:01234567-0123-0123-0123-0123456789abc
```

Alternatively, we could parse the owner reference Kind (i.e. `DaemonSet` and `ReplicationController`) and produce:

```
daemon-set-name:blog
daemon-set-uid:01234567-0123-0123-0123-0123456789abc
```
or

```
replica-set-name:blog
replica-set-uid:01234567-0123-0123-0123-0123456789abc
```

The only downside i can see to the latter approach is having to extend the plugin to recognize new OwnerReference kind's.

Reply:
pod-owner:DaemonSet:blog
pod-owner-uid:DaemonSet:01234567-0123-0123-0123-0123456789abc

IMO

Reply:
We could also construct the name using something like https://godoc.org/github.com/gobuffalo/flect#Dasherize to mitigate the concern over new OwnerReference kinds.

Reply:
On more thought, dasherizing the name would provide unnecessary friction for those crafting registration entries from PodSpec's (they'd have to follow the same convention).  I waffled a bit on `pod-owner` v.s. `pod-owner-name`, but I'm inclined to embrace your proposal.